docs/excel/custom-functions-batching.md

```diff
@@ -27,7 +27,7 @@ To set up batching for your custom functions you'll need to write three main sec
 2. A function to make the remote request when the batch is ready.
 3. Server code to respond to the batch request, calculate all of the operation results, and return the values.
 
-In the following sections you will be shown how to construct the code one example at a time. You'll add each code example to your **functions.ts** file. It's recommended you create a brand new custom functions project using the Yo Office generator. To create a new project see [Get started developing Excel custom functions](../quickstarts/excel-custom-functions-quickstart.md) and use TypeScript instead of JavaScript.
+In the following sections you will be shown how to construct the code one example at a time. You'll add each code example to your **functions.ts** file. It's recommended you create a brand new custom functions project using the [Yeoman generator for Office Add-ins](../develop/yeoman-generator-overview.md) generator. To create a new project see [Get started developing Excel custom functions](../quickstarts/excel-custom-functions-quickstart.md) and use TypeScript instead of JavaScript.
```

Comment:
```suggestion
In the following sections you'll learn how to construct the code one example at a time. You'll add each code example to your **functions.ts** file. It's recommended you create a brand-new custom functions project using the [Yeoman generator for Office Add-ins](../develop/yeoman-generator-overview.md) generator. To create a new project, see [Get started developing Excel custom functions](../quickstarts/excel-custom-functions-quickstart.md) and use TypeScript instead of JavaScript.
```